src/services/formatService.ts

```diff
@@ -56,6 +57,24 @@ export class FormatService {
         return new Intl.DateTimeFormat("en-GB", options).format(date);
     }
 
+    public static formatDateIdentityDocuments (documents?: DocumentDetails[]) {
+        const formattedDocument: any[] = [];
```

Comment:
instead of any type can formattedDocument be of DocumentDetails type?

Reply:
This method has now been removed, thanks. 
I have followed your suggestion here and included an extra field `'formattedExpiryDate'` within the DocumentDetails object that takes in a String and is used to display the expiry date in a readable format when on the Check your answers page